js/apps/system/_admin/aardvark/APP/react/src/views/query/editor/QueryEditorPane.tsx

```diff
@@ -20,24 +21,38 @@ export const QueryEditorPane = () => {
     );
   }
   return (
-    <Box background="white" height="full">
-      <QueryEditorTopBar />
-      <Grid gridTemplateColumns="1.4fr 4px 0.6fr" height="calc(100% - 300px)">
-        <AQLEditor
-          autoFocus
-          resetEditor={resetEditor}
-          value={queryValue}
-          onChange={onQueryValueChange}
-        />
-        <Divider
-          borderLeftWidth="4px"
-          borderColor="gray.400"
-          orientation="vertical"
-        />
-        <QueryOptionsPane />
-      </Grid>
-      <Divider borderColor="gray.400" borderBottomWidth="4px" />
-      <QueryEditorBottomBar />
+    <Box background="white" height="calc(100vh - 300px)">
+      <Resizable
+        style={{ overflow: "hidden" }}
```

Comment:
the handle for resiable doesn't make it obvious that this can be resized; we need a UI affordance to make it more clear

resizable allows for a handleComponent for the same, I'd suggest using that